torch/testing/_internal/distributed/fake_pg.py

```diff
@@ -11,7 +11,7 @@ class FakeStore(dist.Store):
     """
 
 
-def _create_fake_pg(prefix_store, rank, world_size, timeout):
+def _create_fake_pg(common_opts, backend_opts):
```

Comment:
Switched to using the "extended API" as otherwise the options wouldn't get passed to the FakePG constructor

Reply:
I am wondering it this will cause any bc breaking for some tests?